esx_service/cli/vmdkops_admin.py

```diff
@@ -388,8 +389,11 @@ def generate_ls_rows():
         policy = get_policy(metadata, path)
         size_info = get_vmdk_size_info(path)
         created, created_by = get_creation_info(metadata)
+        access = get_access(metadata)
```

Comment:
I plan to change the column names back to "access" and "attach-as". So keep the change as it is to make them consistent.